lib/import/mobility.js

```diff
@@ -0,0 +1,192 @@
+var ArgumentParser = require('argparse').ArgumentParser;
+var mongoose = require('mongoose');
+var csv = require('fast-csv');
+var fs = require('fs');
+
+var config = require('../../config');
+var Mobility = require('../../app/models/mobility');
+
+/**
+ * Convert date from EST to ISO format
+ *
+ * @param{string} date - asdf
+ * @return{date}.
+ */
+function prep_isodate(date) {
+  date = new Date(date);
+  date.setHours(date.getHours());
```

Comment:
I meant line 17 specifically: you set hours to itself?

In any case, I think we should get rid of this function.